### DISCUSSION TITLE
Incorrect value for Generator in wheel metadata.

### DISCUSSION THREAD

**Discussion by domdfcoding** [pradyunsg/sphinx-theme-builder#28] [Potential Issue] [answered]
`sphinx-theme-builder` generates `WHEEL` files in the wheels it builds with an incorrect value for `Generator` of ``flit {version}``

See for instance furo's wheel: https://www.wheelodex.org/projects/furo/#wheel

The code that creates the file is here:

https://github.com/pradyunsg/sphinx-theme-builder/blob/aff12ad5b472315496b6bc70272d02d0cda8fa19/src/sphinx_theme_builder/_internal/distributions.py#L135-L144

It should say, for example, `sphinx-theme-builder 1.2.3`.

**Comment by pradyunsg** [comment]
Ahahaahahahahaha, indeed.

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/sphinx-theme-builder/commit/2a7d3f5ad2573faffa847bfcfcc681c5438e5f6d fixed this.
